lightkurve/tests/test_correctors.py

```diff
@@ -71,3 +72,15 @@ def test_sff_corrector():
     assert_almost_equal(4*sff.s, arclength, decimal=2)
     assert_almost_equal(sff.interp(sff.s), correction, decimal=3)
     assert_array_equal(time, klc.time)
+
+@pytest.mark.remote_data
+def test_pld_corrector():
+    # download tpf data for a target
+    target = 247887989
+    tpf = search_targetpixelfile(target).download()
+
+    # instantiate PLD corrector object
+    pld = PLDCorrector(tpf)
+
+    # produce a PLD-corrected light curve
+    corrected_lc = pld.correct()
```

Comment:
I feel there should be a few more tests, given the amount of code we're adding. We need tests for 

`pld.diagnose()`
`pld.correct(mask=X)`
`pld.correct(difference number of components)`
`tpf = search_targetpixelfile('EPIC NUMBER').download()`
`tpf = search_targetpixelfile('TIC NUMBER').download()`

This would be a nice place to add some scope lightkurves where we know the ground truth and do some test to see we retrieve the correct answer!

Reply:
`pld.correct(aperture_mask='all')`
`pld.correct(aperture_mask=threshold')`
`pld.correct(aperture_mask=tiny)`